src/cache.rs

```diff
@@ -61,19 +61,9 @@ impl Cache {
 
     /// Download the archive
     fn download(&self) -> Result<Vec<u8>, TealdeerError> {
-        let mut handle = CurlEasy::new();
-        try!(handle.url(&self.url));
-        try!(handle.follow_location(true));
-        try!(handle.fail_on_error(true));
-        let mut buf = Vec::new();
-        {
-            let mut transfer = handle.transfer();
-            try!(transfer.write_function(|data| {
-                buf.extend_from_slice(data);
-                Ok(data.len())
-            }));
-            try!(transfer.perform());
-        }
+        let mut resp = reqwest::get(&self.url)?;
+        let mut buf: Vec<u8> = vec![];
+        resp.copy_to(&mut buf)?;
```

Comment:
The `unused_results` linter complains here. To avoid, use `let _bytes = resp.copy_to(&mut buf)?;`.

Reply:
What do you think about logging it instead? It's not quite the same, but it is getting rid of the linter warning and shouldn't hurt anyway.

Reply:
Ah, yeah, why not. Something like "Downloaded {} bytes".

Reply:
I worded it slightly differently, "{} bytes downloaded", but the result is the same ^^